lib/vagrant-service-manager/command.rb

```diff
@@ -221,6 +230,28 @@ def print_vagrant_box_version(script_readable = false)
           end
         end
       end
+
+      def restart_service(service)
+
+        if SCCLI_SERVICES.include? service
+          # since sccli understand k8s
+          service = "k8s" if service == "kubernetes"
+          # TODO : Handle the case where user wants to pass extra arguments to OpenShift service
+          command = "sudo sccli #{service}"
+        else
+          command = "sudo systemctl restart #{service}"
```

Comment:
> I think we should check the validity of service here. If service is not valid (suppose someone fire vagrant service-manager restart abc) then will raise error as Invalid service.

Thats handled by `systemd` and `systemctl` show the correct error message which we relay via plugin. @budhrg : Try testing the PR with different scenarios ?